engine/apps/metrics_exporter/metrics_cache_manager.py

```diff
@@ -40,56 +40,59 @@ def get_default_states_diff_dict():
         return default_dict
 
     @staticmethod
-    def update_integration_states_diff(metrics_dict, integration_id, previous_state=None, new_state=None):
-        metrics_dict.setdefault(integration_id, MetricsCacheManager.get_default_states_diff_dict())
+    def update_integration_states_diff(metrics_dict, integration_id, service_name, previous_state=None, new_state=None):
+        state_per_service = {service_name: MetricsCacheManager.get_default_states_diff_dict()}
+        metrics_dict.setdefault(integration_id, state_per_service)
```

Comment:
Makes sense. It was used for a bulk update cache, and the incoming data there was grouped by integration. We don't use a bulk update anymore and the metrics_dict always contains only one integration with information about one alert group, so there is no difference there, but since we have it, it's better that it works correctly in different cases 👍